node/jaeger/src/spans.rs

```diff
@@ -277,7 +277,7 @@ impl Span {
 	}
 
 	/// Derive a child span from `self`.
-	pub fn child(&self, name: &'static str) -> Self {
+	pub fn child(&self, name: &str) -> Self {
```

Comment:
The 'static lifetime bind is unnecessary here, decided to remove it, because it prevents you from generally doing simple stuff like.

span.child(any_dynamic_string.as_str()).